src/phoenix/experimental/callbacks/langchain_tracer.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+from uuid import uuid4
+
+from langchain.callbacks.tracers.base import BaseTracer
+from langchain.callbacks.tracers.schemas import Run
+
+from phoenix.trace.schemas import Span, SpanContext, SpanKind, SpanStatusCode
+
+
+class OpenInferenceTracer(BaseTracer):  # type: ignore
+    def __init__(self) -> None:
+        self.spans: List[Span] = []
+        self.trace_id = uuid4()
+        super().__init__()
+
+    def _persist_run(self, run: Run) -> None:
```

Comment:
we need some sort of persistence abstraction here so we can mount this tracer in the callback manager and extract a file of the spans. Were you planning on doing this as a follow-up?

Reply:
revised to use existing Tracer class